index.html

```diff
@@ -185,39 +223,65 @@ <h2>Languages</h2>
+        <li class="js">
+          <img class="mobile-icon" src="./Images/Frame9.png" alt="js">
+          <img class="js-shape" src="./Images/JS.png" alt="JS icon">
+          <h3 class="js-txt">JavaScript</h3>
+        </li>
+        <li class="html">
+          <img class="mobile-icon" src="./Images/Frame10.png" alt="html">
+          <img class="html-shape" src="./Images/html.png" alt="HTML icon">
+          <h3 class="html-txt">HTML</h3>
+        </li>
+        <li class="css">
+          <img class="mobile-icon" src="./Images/Frame11.png" alt="css">
+          <img class="css-shape" src="./Images/css.png" alt="CSS icon">
+          <h3 class="css-txt">CSS</h3>
+        </li>
       </ul>
 
-      <ul class="framework">
+      <ul class="framework ">
         <li>
           <h2>Frameworks</h2>
         </li>
         <li><img src="./Images/Union0.png" alt="downward icon"></li>
       </ul>
 
+      <img class="separator" src="./Images/Separator.png" alt="Bottom Separator">
+
       <ul class="skills">
         <li>
           <h2>Skills</h2>
         </li>
         <li><img src="./Images/Union0.png" alt="downward icon"></li>
       </ul>
 
+      <img class="separator" src="./Images/Separator.png" alt="Bottom Separator">
     </div>
   </section>
 
-  <section class="form-background">
-    <h2 class="form-heading">Contact me</h2>
-    <p class="form-content">If you have an application you are interested in developing, a feature that you need built
-      or a project that needs coding. I’d love to help with it.</p>
-    <form class="contact-form" action="https://formspree.io/f/mvolvljr" method="POST">
-      <input class="name" type="text" name="firstname" maxlength="30" placeholder="Please enter your name " required>
-      <input class="email" type="email" name="email" placeholder="Please enter your email" required>
-      <textarea class="message" name="message" maxlength="500" placeholder="Write your message here"
-        required></textarea>
-      <button class="submit" type="submit">Get it touch</button>
-    </form>
+  <section class="main-contact port-form">
+    <div class="form-background">
+      <div class="form-fix">
+        <h2 class="form-heading">Contact me</h2>
+        <p class="form-content">If you have an application you are interested in developing, a feature that you need
+          built
+          or a project that needs coding. I’d love to help with it.</p>
+        <form class="contact-form" action="https://formspree.io/f/mvolvljr" method="POST">
+          <ul>
+            <li><input class="name" type="text" name="firstname" maxlength="30" placeholder="Please enter your name "
+                required></li>
+            <li><input class="email" type="email" name="email" placeholder="Please enter your email" required></li>
+            <li>
+              <textarea class="message" name="message" maxlength="500" placeholder="Write your message here"
+                required></textarea>
```

Comment:
- [ ] [OPTIONAL] It appears that the new changes introduced broke the design a little bit. Kindly make the width of the text area to match the chosen Figma design.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/188196756-cf0f3fcb-0871-47c4-97a3-3dad28265d82.png)